shared/cfg/guest-hw.cfg

```diff
@@ -352,6 +357,10 @@ variants image_backend:
         nfs_mount_src = localhost:/mnt/nfs_export
         setup_local_nfs = yes
         export_options = rw
+    - nvme_direct:
+        storage_type = nvme
```

Comment:
nvme-direct instead of nvme?
we introduce nvme_direct backend, then, I think there might be some other nvme backends,  we might need storage_type to distinguish between them.

Reply:
Please refer to https://github.com/avocado-framework/avocado-vt/pull/2608/files#r447538292
Thanks.